tests/integration.sh

```diff
@@ -998,8 +1001,15 @@ function test_send() {
     log 'sending funds from "a" to the Receiver, with message to the Receiver'
     local receiver_msg='{"increment":{}}'
     receiver_msg="$(base64 <<<"$receiver_msg")"
-    local send_message='{"send":{"recipient":"'"$receiver_addr"'","amount":"400000","msg":"'$receiver_msg'"}}'
+    
+    if [ "$skip_register_receiver" = "true" ]; then
+        send_message='{"send":{"recipient":"'"$receiver_addr"'","recipient_code_hash":"'"$receiver_hash"'","amount":"400000","msg":"'$receiver_msg'"}}'
+    else
+        send_message='{"send":{"recipient":"'"$receiver_addr"'","amount":"400000","msg":"'$receiver_msg'"}}'
+    fi
+
```

Comment:
Please add `local send_message` before the `if`. Bash behaves like javascript here, if you don't say it's local, it's a global.

Reply:
Done